.github/workflows/ci-docs.yaml

```diff
@@ -46,7 +46,13 @@ jobs:
   e2e-tests:
     strategy:
       matrix:
-        os: [[self-hosted, macos, amd64, 13], [self-hosted, macos, amd64, 12.6], [self-hosted, macos, arm64, 13], [self-hosted, macos, arm64, 12.6]]
+        os: 
+          [
+            [self-hosted, macos, amd64, 13, test], 
```

Comment:
Have the current runners already been tagged with "test"? Will the e2e test run for this PR?

Reply:
Not yet, as we don't know when this PR will be merged (don't want the runners to be unavailable), I will change the labels on the runners right before I merge it. Note that when this PR is merged, any current PRs will also need to update their branch to have the right labels/run CI.